src/index.ts

```diff
@@ -7,9 +7,13 @@ export = (app: Application) => {
     const user = head.user.login;
     const repo = head.repo.name;
 
-    const comment = `Thanks for making a pull request to JupyterLab!
+    // Get binder url
+    const config = await context.config('config.yml', { 'binder_url': 'https://mybinder.org' })
+    const binder = config.binder_url
 
-To try out this branch on [binder](https://mybinder.org), follow this link: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/${user}/${repo}/${ref}?urlpath=lab-dev)`
+    const comment = `Thanks for making a pull request to ${repo}!
```

Comment:
This could be easily read from a config file.

Reply:
I think having a display name for the repo would be nice. For instance, I'd like the JupyterLab bot to say "Thanks for making a pull request to JupyterLab", rather than "Thanks for making a pull request for jupyterlab". I'd imagine others would want that as well.